META.json

```diff
@@ -1,16 +1,18 @@
 {
    "name": "pgtelemetry",
    "abstract": "Useful monitoring and metrics views for PostgreSQL",
-   "version": "1.4.0",
-   "maintainer": [ "Chris Travers <christravers@adjust.com>" ],
+   "version": "1.6.0",
```

Comment:
More precisely the version number is `1.6`. See here:
https://github.com/adjust/pg-telemetry/blob/master/pgtelemetry.control#L1

But I'm not sure if there is difference in PGXN.